src/components/FullTags/TagsCombinaisonsForm.vue

```diff
@@ -0,0 +1,378 @@
+<template>
+    <form class="tag-selection-form" @submit="prevent">
+        <legend>Images with tags form</legend>
+        <!-- Quand l'utilisateur entre un tag avec enter, on l'ajoute, qu'il existe ou non. -->
+        <input type="text" class="tag-filter-input" placeholder="SEARCH TAG TO FILTER" @focus="show_tags_options = true"
+            @input="updateSearchTag($event)" @keyup.enter="this.selected_tags.push(this.selected_tags[0])" />
```

Comment:
With data binding, I think you simplify the addTags method + the formatted string in your computed part.